### DISCUSSION TITLE
Welcome to recipe-chooser Discussions!

### DISCUSSION THREAD

**Discussion by FranciscoPedreira** [FranciscoPedreira/recipe-chooser#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joaopiopedreira** [comment]
Que é lá isto pá? 👍

**Comment by FranciscoPedreira** [comment]
Ahahah é uma cena nova do GitHub!!
